budgetportal/tests/test_department_page.py

```diff
@@ -143,6 +143,7 @@ def test_missing_budget_dataset(self):
         response = c.get("/2018-19/national/departments/the-presidency/")
 
         self.assertContains(response, "Data not available")
+        self.assertNotContains(response, "Budget (Main appropriation) 2019-20")
```

Comment:
this text should match the text used here https://github.com/vulekamali/datamanager/pull/347/commits/45b089bb48743c289c436f7d1d733e9d60b731ac#diff-bbdee1689dc68facf987f1c5ed2780a12cd528ccfe89315441b82028448240a4R159